11ty/definitions/r-word.md

```diff
@@ -0,0 +1,33 @@
+---
+title: R-word
+slug: r-word
+defined: true
+speech: noun
+flag:
+  level: avoid
+  text: ableist slur
```

Comment:
Add 

`reading:
- text: "Why Use of the R-Word Needs to Stop"
  href: https://www.verywellfamily.com/what-is-the-r-word-3105651
- text: "End the R word"
  href: https://www.sonc.org/EndtheRWord